src/index.js

```diff
@@ -1,14 +1,47 @@
-import _ from 'lodash';
+// import _ from 'lodash';
 import './style.css';
 
-function component() {
-    const element = document.createElement('div');
-  
-    // Lodash, now imported by this script
-    element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-    element.classList.add('hello');
-  
-    return element;
+// add to do list
+const toDoList = [];
+
+const taskOne = {
+  description: 'make the bed',
+  completed: false,
+  index: 0,
+};
+
+toDoList.push(taskOne);
+
+const taskTwo = {
+  description: 'walk the dog',
+  completed: false,
+  index: 1,
+};
+toDoList.push(taskTwo);
```

Comment:
- [ ] Great job for making this effort, but there's a better way of doing this. You can create an array of objects with each object representing a todo list. and then iterate over the array to populate your todo lists on page load. Take a look :point_down: 

const todoList = [
  {
    description: 'make the bed',
    completed: false,
    index: 0,
  },
  {
     description: 'walk the dog',
    completed: false,
    index: 1,
  },
  {
    etc......
  }
];